platform/ios/Integration Tests/MGLCameraTransitionTests.mm

```diff
@@ -323,11 +323,7 @@ - (void)testFlyToCameraInDelegateMethod {
     // Should take MGLAnimationDuration
     [self.mapView setCenterCoordinate:target zoomLevel:zoomLevel animated:YES];
 
-    // Run the loop, so the camera can fly to the new camera
-    while (runloop) {
```

Comment:
This seems like a good change. Coming in with fresh eyes, this test feels pretty complicated. removing this while loop and flag reduces the internal coupling (and of course fixes a stall-out on iOS 9)